### PR TITLE
[Robot] Test to validate cohort and PE lookup to same program

### PR DESCRIPTION
**Pull request by rmonica08**
1. Added test to validate cohort and PE lookup to same program
2. Fixed verify details keyword
3. Updated locators

W-042769

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
- [ ] Place holder data is incorporated (Refer to [Case Management Sample Data document](https://quip.com/cbFcAPJF8t0z))
- [ ] Base axe-core JEST test included for any new LWC (No critical violations)
- [ ] CRUD/FLS is enforced in Apex
- [ ] Permission sets are updated to account for CRUD/FLS for new object metadata
- [ ] All modified classes are unit tested (Apex) at 100% coverage
- [ ] UX approval or UX not necessary
- [ ] PR contains draft release notes
- [ ] Attach work item to the pull request with [Lurch](https://salesforce.quip.com/50ZRA5LEWVzH) `**Lurch:attach W-000000` or `**Lurch:add`
- [ ] All **acceptance criteria** have been met
    - [ ] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [ ] QE story level testing completed

### CODE REVIEW

File: robot/pmm/tests/browser/ServiceDeliveries/new_service_delivery.robot [hunk 1]
```diff
@@ -7,7 +7,7 @@ Library        cumulusci.robotframework.PageObjects
 Suite Setup     Run Keywords
 ...             Open Test Browser
 ...             Setup Test Data
-Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+#Suite Teardown  Capture Screenshot and Delete Records and Close Browser
```
Comment (gaganpsandhu) [inline robot/pmm/tests/browser/ServiceDeliveries/new_service_delivery.robot:10]: Add Teardown method back

File: robot/pmm/tests/browser/Program/pe_quickaction.robot [hunk 2]
```diff
@@ -25,10 +24,17 @@ Setup Test Data
     ${result_date} =      Get Current Date                result_format=%Y-%m-%d
     Set suite variable    ${result_date}
```
Comment (gaganpsandhu) [inline robot/pmm/tests/browser/Program/pe_quickaction.robot:26]: remove this blank line

File: robot/pmm/tests/browser/ServiceDeliveries/new_service_delivery.robot (hunk 1 above)
Comment (gaganpsandhu) [inline robot/pmm/tests/browser/ServiceDeliveries/new_service_delivery.robot:10]: Add Teardown method

Comment (rmonica08) [reply robot/pmm/tests/browser/ServiceDeliveries/new_service_delivery.robot]: Added it back

File: robot/pmm/tests/browser/Program/pe_quickaction.robot (hunk 2 above)
Comment (rmonica08) [reply robot/pmm/tests/browser/Program/pe_quickaction.robot]: Removed

File: robot/pmm/tests/browser/ServiceDeliveries/new_service_delivery.robot (hunk 1 above)
Comment (rmonica08) [reply robot/pmm/tests/browser/ServiceDeliveries/new_service_delivery.robot]: Looks like the same file as above, added suite teardown back.

**Pull request merged**